tiny11creator.ps1

```diff
@@ -1,44 +1,56 @@
 #Defining preferences variables
+Write-Output "Loading configuration from config.json..."
 $config = (Get-Content "config.json" -Raw) | ConvertFrom-Json
 $wantedImageName = $config.WantedWindowsEdition
 $unwantedProvisionnedPackages = $config.ProvisionnedPackagesToRemove
 $unwantedWindowsPackages = $config.WindowsPackagesToRemove
+$pathsToDelete = $config.PathsToDelete
 
 #Defining system variables
+Write-Output "Creating needed variables..."
 $rootWorkdir = "c:\tiny11\"
 $isoFolder = $rootWorkdir + "iso\"
 $installImageFolder = $rootWorkdir + "installimage\"
 $bootImageFolder = $rootWorkdir + "bootimage\"
 $isoPath = "c:\windows11.iso"
+$yes = ("O" | choice)[1] #this variable gets the "y" from "yes" (or any other letter it the "yes" word doesn't start with "y" in your language)
```

Comment:
This does not work on my PowerShell 5.1

ERROR: The file is either empty or does not contain the valid choices.

Reply:
Thanks for you feedback, I was indeed hesitant to include this hacky detection of a localized "yes/no". I'll try it on PowerShell 5.1.
For information I'm very new to PowerShell in general.
Also choice is a cmd command, maybe that's why?

Reply:
Oh I figured the issue...
I'm just answering "O" to end the choice process and get back to the rest of the PowerShell script.
But "O" works on my french computer for "Oui" but can't work on other languages.
If you have an idea it would be nice otherwise I'll test some workarounds and patch it.
For now, you can just store in this variable the first letter of the word "yes" in your language.
It's only useful for the takeown /d command iirc.

Reply:
@tommybuck patched thanks to this stackoverflow answer [https://stackoverflow.com/a/56743503](https://stackoverflow.com/a/56743503) :)